### PR TITLE
fixed broken links

### PR DESCRIPTION
**Pull request by eli9701**
fixed broken albums link and change the sask artists link from a list of other sask artists to the acctual sask artists page

### CODE REVIEW

**Comment by eli9701** [comment]
fixes #432

**Comment by eli9701** [comment]
fixes #433

**Comment by eli9701** [comment]
wrong one the first time

File: www/artists/jmitchell2/album.html [hunk 1]
```diff
@@ -14,7 +14,7 @@ <h1>
             Joni's Album Art &#9829;
         </h1>
         <div id="nav">
-            <a href="index.html">Home</a> <a href="album.html">Album Artworks</a> <a href="paintings.html">Paintings</a> <a href="discography.html">Discography</a> <a href="../">Saskatchewan Artists</a> <a href="http://www.jonimitchell.com">Joni Mitchell's Site</a>
+            <a href="index.html">Home</a> <a href="album.html">Album Artworks</a> <a href="paintings.html">Paintings</a> <a href="discography.html">Discography</a> <a href="https://saskartists.ca/">Saskatchewan Artists</a> <a href="http://www.jonimitchell.com">Joni Mitchell's Site</a>
```
Comment (gemohandy) [inline www/artists/jmitchell2/album.html]: Avoid absolute links to SaskArtists.

File: www/artists/jmitchell2/discography.html [hunk 2]
```diff
@@ -11,7 +11,7 @@ <h1>
                 Discography ♥
             </h1>
             <div id="nav">
-                <a href="index.html">Home</a> <a href="album.html">Album Artworks</a> <a href="paintings.html">Paintings</a> <a href="discography.html">Discography</a> <a href="../">Saskatchewan Artists</a> <a href="http://www.jonimitchell.com">Joni Mitchell's Site</a>
+                <a href="index.html">Home</a> <a href="album.html">Album Artworks</a> <a href="paintings.html">Paintings</a> <a href="discography.html">Discography</a> <a href="https://saskartists.ca/">Saskatchewan Artists</a> <a href="http://www.jonimitchell.com">Joni Mitchell's Site</a>
```
Comment (gemohandy) [inline www/artists/jmitchell2/discography.html]: Avoid absolute links to SaskArtists

File: www/artists/jmitchell2/index.html [hunk 3]
```diff
@@ -11,7 +11,7 @@ <h1>
             Joni Mitchell &#9829;
         </h1>
         <div id="nav">
-            <a href="index.html">Home</a> <a href="album.html">Album Artworks</a> <a href="paintings.html">Paintings</a> <a href="discography.html">Discography</a> <a href="../">Saskatchewan Artists</a> <a href="http://www.jonimitchell.com">Joni Mitchell's Site</a>
+            <a href="index.html">Home</a> <a href="album.html">Album Artworks</a> <a href="paintings.html">Paintings</a> <a href="discography.html">Discography</a> <a href="https://saskartists.ca/">Saskatchewan Artists</a> <a href="http://www.jonimitchell.com">Joni Mitchell's Site</a>
```
Comment (gemohandy) [inline www/artists/jmitchell2/index.html]: Avoid absolute links to SaskArtists.

File: www/artists/jmitchell2/paintings.html [hunk 4]
```diff
@@ -14,7 +14,7 @@ <h1>
             Joni's Paintings &#9829;
         </h1>
         <div id="nav">
-            <a href="index.html">Home</a> <a href="albums.html">Album Artworks</a> <a href="paintings.html">Paintings</a> <a href="discography.html">Discography</a> <a href="../">Saskatchewan Artists</a> <a href="http://www.jonimitchell.com">Joni Mitchell's Site</a>
+            <a href="index.html">Home</a> <a href="album.html">Album Artworks</a> <a href="paintings.html">Paintings</a> <a href="discography.html">Discography</a> <a href="https://saskartists.ca/">Saskatchewan Artists</a> <a href="http://www.jonimitchell.com">Joni Mitchell's Site</a>
```
Comment (gemohandy) [inline www/artists/jmitchell2/paintings.html]: Avoid absolute links to SaskArtists. The album link looks good, though.

**Pull request merged**